decimal.go

```diff
@@ -422,6 +423,50 @@ func (d Decimal) Round(places int32) Decimal {
 	return ret
 }
 
+// RoundFair rounds the decimal to places decimal places.
+// If places < 0, it will round the integer part to the nearest 10^(-places).
+//
+// Unlike Round(), in the case of a a tie where the resulting decimal place
+// equals 0.5, this function will round up for odd numbers and down for
+// even numbers. Negative values are treated symmetrically.
+//
+// Example:
+//
+// 	   NewFromFloat(5.5).Round(0).String() // output: "6"
+// 	   NewFromFloat(8.5).Round(0).String() // output: "8"
+//
+func (d Decimal) RoundFair(places int32) Decimal {
+	shift := big.NewInt(abs(int64(places)-int64(d.exp)) - 1)
```

Comment:
Please update this to use `math.Abs` rather than implementing our own `abs()` function in this library.

For example:
```go
shift := big.NewInt(int64(math.Abs(float64(places) - float64(exp)) - 1))
```